tutorials/math/matrices_and_transforms.rst

```diff
@@ -308,36 +308,25 @@ as well as :ref:`Spatial.to_local() <class_Spatial_method_to_local>` and :ref:`S
 Scale
 ~~~~~
 
-A matrix can be scaled too. Scaling will multiply the basis vectors by a
-vector (X vector by x component of the scale, Y vector by y component of
-the scale). It will leave the origin alone:
+A matrix can be scaled too. Scaling will multiply the X and Y vectors of the
+matrix by the x and y components of another vector (`X * x`, `Y * y`). It will
```

Comment:
Just a nitpick, should we capitalize "x" and "y" in "x and y components"? If not, we might want to write them as ``x`` and ``y`` instead (as if they're member variables in a type).

Reply:
> Just a nitpick, should we capitalize "x" and "y" in "x and y components"? If not, we might want to write them as x and y instead (as if they're member variables in a type).

The capitalization was to differentiate between the member variables and the vectors, but we can use code blocks instead :) Do you think the `X * x` bit be X * `x` instead as well?

> Also, if I recall corrctly, reStructuredText recommends using two backticks instead of one for inline code.

Oop, markdown on the brain. Good catch!